stable/postgresql/values.yaml

```diff
@@ -3,7 +3,7 @@ image: "postgres"
 ## postgres image version
 ## ref: https://hub.docker.com/r/library/postgres/tags/
 ##
-imageTag: "9.6.2"
+imageTag: "10.5"
```

Comment:
You are changing the postgres version here, which seems out of the scope of this PR.

Reply:
I agree. But then again so should changing the verison of the helm chart.

Happy to revert, but frankly:
- I'm not sure anymore what should be the scope and what not
- I have tested the chart with this new version and does work OK.

I'd kindly ask that this PR finally gets merged, since it works and it's an improvement given current situation either way.

Reply:
It would be healthier to separate concerns and do two differents PRs. A lot of people are cherry-picking commits and may see their postgres version change without realizing (no hint from commit message).

Although I am happy with both changes. :)

Reply:
What if we just change the commit message then?

Reply:
I'm personally against this, but let's see other people's point of view. It is easy to do two different PRs for different concerns.